assets/scss/generic-overrides.scss

```diff
@@ -64,7 +63,7 @@
     }
 
     .nhsuk-pagination__link:hover {
-      box-shadow: 0 0 0 nhsuk-spacing(1) $color_nhsuk-warm-yellow inset;
+      box-shadow: 0 0 0 nhsuk-spacing(1) $nhsuk-focus-color inset;
```

Comment:
I would suggest consistency in variable names here `$nhsuk-focus-color`, there `$color_nhsuk-white `